dlp/snippets/deid/deid_simple_word_list.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deid
+
+// [START dlp_deidentify_simple_word_list]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// deidentifyWithWordList matches against a custom simple word list to de-identify sensitive
+// data based on the input
+func deidentifyWithWordList(w io.Writer, projectID, input string, infoTypeName string, wordList []string) error {
+	// projectID := "my-project-id"
+	// input := "Patient was seen in RM-YELLOW then transferred to rm green."
+	// wordList := []string{"RM-GREEN", "RM-YELLOW", "RM-ORANGE"}
+	ctx := context.Background()
+	// Initialize a client once and reuse it to send multiple requests. Clients
```

Comment:
nit: put in another line break after the `ctx := context.Background()` line. Otherwise this looks like a big wall of text.

Reply:
fixed it.